### PR TITLE
Ecalpedestal pcl g6g1treat 92x

### PR DESCRIPTION
**Pull request by argiro**
Modify the Ecal pedestal PCL workflow so that G6 and G1 pedestals are copied from the HLT label. The HLT will always contain the most updated measurement of G6 and G1 peds.

92X version of #19793

### CODE REVIEW

**Comment by arunhep** [comment]
backport of #19793

**Comment by arunhep** [comment]
please test

File: Calibration/EcalCalibAlgos/test/ecalPedestalPCL_step2_cfg.py [hunk 1]
```diff
@@ -32,7 +32,8 @@
 
 # Other statements
 from Configuration.AlCa.GlobalTag import GlobalTag
-process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_data', '')  
+#process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_data', '')  
+process.GlobalTag = GlobalTag(process.GlobalTag, '92X_dataRun2_Express_v2_withECALPedHLT', '')
```
Comment (franzoni) [inline Calibration/EcalCalibAlgos/test/ecalPedestalPCL_step2_cfg.py]: hello @argiro 
please don't hard code this GT which we provided you for your local initial tests
run2_data does provide you with the record/label you need, see : 
https://cms-conddb-int.cern.ch/cmsDbBrowser/diff/Prod/gts/92X_dataRun2_v2/92X_dataRun2_v5

**Comment by arunhep** [comment]
please test

**Comment by arunhep** [comment]
+1

**Comment by franzoni** [comment]
please test

**Comment by franzoni** [comment]
(only for completeness, Stefano's last commit does not change anything nor does it get tested )

**Comment by davidlange6** [comment]
+1

**Pull request merged**